Gemfile

```diff
@@ -44,6 +44,9 @@ gem "mail-notify", ">= 1.0.3"
 # do not rely on host's timezone data, which can be inconsistent
 gem "tzinfo-data"
 
+# serialization gem that offers more features than active model serializer
+gem "blueprinter"
```

Comment:
👍 I haven't used this one before. I'll check it out!

Reply:
Originally I thought about active model serializers and then I remembered how hard sometimes was to see the output of the serialization - and then found this little gem - pretty straightforward and easy to use